src/components/screen/Oauth/modules/ServerSwitcher.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <div class="server-switcher">
+    <v-label label="Choose a server" />
+    <div class="server-switcher-container">
+      <v-select
+        v-model="currentServer"
+        :options="options"
+        class="server-switcher-selector"
+      />
+      <v-button
+        is-inline
+        :disabled="disabled"
+        :is-loading="isLoading"
+        class="server-switcher-button"
+        @click="onServerSwitch"
+      >
+        Set
+      </v-button>
+    </div>
+  </div>
+</template>
+
+<script>
+import VSelect from '@endpass/ui/kit/VSelect';
+import VLabel from '@endpass/ui/kit/VLabel';
+import VButton from '@endpass/ui/kit/VButton';
+
+import { connectStore } from '@/store';
+
+export default {
+  name: 'PopupModeSwitcher',
+
+  connectStore,
+
+  props: {
+    options: {
+      type: Array,
+      default: () => [],
+    },
+  },
+
+  data() {
+    return {
+      currentServer: this.$options.connectStore.server,
```

Comment:
don't need to get `.server`, 
you can just check window.location for check current state